functions/src/custom/totals.ts

```diff
@@ -0,0 +1,76 @@
+import * as functions from "firebase-functions";
+import * as Sentry from "@sentry/node";
+import logger from "../services/logging";
+import { firestore } from "../admin/admin";
+import { environment } from "../environment";
+
+const event_collection = environment.FIRESTORE_EVENT_COLLECTION as string;
+
+export const computeCollectionTotals = firestore.listCollections().then(async (collections) => {
+  const total: Record<string, number> = {};
+  const collectionTotalRef = firestore.collection("total").doc("collection_total");
```

Comment:
@NickBurnett Lets abstract away the fixed collection / document names into Doppler env variables like everywhere else.

Reply:
@NickBurnett Hey, any update on the updated Doppler values so I can add them into the PR? Thanks!

Reply:
@harshasrikara For some reason, I just saw this ?? But I actually don't have access to Doppler to abstract anything away, but I'll dm @WillieCubed to see if he can restore my access.

Reply:
@NickBurnett there's a how to guide for Doppler, it has a self serve invite link

Reply:
I added `FIRESTORE_TOTALS_COLLECTION = total` and `FIRESTORE_TOTALS_DOC = collection_total` into Doppler.

Reply:
@Stevenmqnguyen Just added these Doppler values in the latest commit. Can you add a Doppler value for the event document as well under the totals collection?

Reply:
@legitatx I added `FIREBASE_TOTALS_EVENT_DOC = totals` to Doppler.